cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -502,6 +502,10 @@ mkConsensusTracers trSel verb tr nodeKern fStats = do
               traceI trmet meta "submissions.rejected.count" =<<
                 STM.modifyReadTVarIO tSubmissionsRejected (+ ptxcRejected processed)
 
+            TraceLabelPeer _ TraceTxInboundTerminated -> return ()
+            TraceLabelPeer _ (TraceTxInboundCanRequestMoreTxs _) -> return ()
+            TraceLabelPeer _ (TraceTxInboundCannotRequestMoreTxs _) -> return ()
```

Comment:
@coot Mainly I wanted to know if something should be done here.

Reply:
No, I don't think we need to do anything here.